server/controllers/dishes/getDishById.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response, NextFunction } from 'express';
+import { Dish } from '../../models';
+import { StatusCodes } from '../../utils/enum';
+import { CustomError } from '../../utils';
+
+const getDishById = async (req: Request, res: Response, next: NextFunction) => {
+  const { id } = req.params;
```

Comment:
Destructure the id in the try statement's scoop. Think what will happen when the user doesn't send an id in the params.

Reply:
okay also done